### PR TITLE
Translatable labels for payment-provider-rules

### PR DESCRIPTION
**Pull request by nehlsen**
Added translatable labels for payment-provider-rules

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #...   

BEFORE
coreshop-add-localized-label-to-payment-provider-rules-before

AFTER
coreshop-add-localized-label-to-payment-provider-rules-after

### CODE REVIEW

File: src/CoreShop/Bundle/CoreBundle/Migrations/Version20230829163505.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+final class Version20230829163505 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/CoreBundle/Migrations/Version20230829163505.php:37]: can wee add checks if the table already exists?

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]
lovely work, just the migration and then we are good to go.

Thank you very much @nehlsen

File: src/CoreShop/Bundle/CoreBundle/Migrations/Version20230829163505.php (hunk 1 above)
Comment (nehlsen) [reply src/CoreShop/Bundle/CoreBundle/Migrations/Version20230829163505.php:37]: Thank You!
I have added a check. does it suffice?

Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/CoreBundle/Migrations/Version20230829163505.php:37]: Can you rather do something like

`if (!$schema->hasTable('coreshop_payment_provider_rule_translation') {return}`

That way the add constraint can also not fail anything.

Comment (nehlsen) [reply src/CoreShop/Bundle/CoreBundle/Migrations/Version20230829163505.php:37]: sure thing :)

**Review by dpfaffenbauer — APPROVED** [review]

**Comment by dpfaffenbauer** [comment]
thanks a lot @nehlsen

**Pull request merged**